addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -27,8 +27,11 @@ def default_get(self, field_list):
     department_id = fields.Many2one('hr.department', "Department", compute='_compute_department_id', store=True, compute_sudo=True)
     encoding_uom_id = fields.Many2one('uom.uom', compute='_compute_encoding_uom_id')
 
+    # Timer Fields
     timesheet_timer_start = fields.Datetime("Timesheet Timer Start", default=None)
 
+    timer = fields.Boolean(string="", compute="_compute_timer", store=False, help="This field is used for timer_toggle_button widget")
```

Comment:
The help message is not very understandable for a common human. Could you adapt ?